packages/squiggle-lang/src/rescript/utility/Sparklines.res

```diff
@@ -0,0 +1,28 @@
+// Port of Sindre Sorhus' Sparkly to Rescript
+// reference implementation: https://github.com/sindresorhus/sparkly
+// Omitting rgb "fire" style, so no `chalk` dependency
+// Omitting: NaN handling, special consideration for constant data.
+
+let create = (
+  numbers: array<float>,
+  ~minimum=?,
+  ~maximum=?,
+  ()
+) => {
+  let ticks = [`▁`, `▂`, `▃`, `▄`, `▅`, `▆`, `▇`, `█`]
+
+  let minimum = E.O.default(Js.Math.minMany_float(numbers), minimum)
```

Comment:
Is there any reason why you use Js.Math.minMany_float instead of ``E.A.Float.min`?

Reply:
not really.

Reply:
I also just noticed that ``minimum`` is never used in this function

Reply:
you're right! now that I cleaned out some case handling. good catch.